emissionscheck_alb/baseline_model.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+
```

Comment:
do all your imports here, it makes the code more readable (and once the script gets longer, you have a far better overview over what you're importing)
```suggestion
    from sklearn.pipeline import make_pipeline
    from sklearn.preprocessing import MinMaxScaler
    from sklearn.metrics import roc_auc_score, plot_confusion_matrix, classification_report
    from sklearn.model_selection import cross_val_score
    from sklearn.linear_model import LogisticRegression
    from sklearn.svm import SVC
```

Reply:
Done!